src/reducetensor.cpp

```diff
@@ -38,6 +38,11 @@
 #include <ostream>
 #include <iostream>
 
+// although gfx1030 supports buffer instructions,but it does not work properly when we using
+// corresponding llvm intrinsic functions
+// so using those llvm intrinsic functions is disabled on gfx1030
```

Comment:
Please remove the comment.

Reply:
This comment and definition refers to the same thing in src/solver/implicitgemm_util.hpp.  Here Imy codes does not use the definition in src/solver/implicitgemm_util.hpp because I am not sure if it is suitable to include this header file.

Reply:
The comment is incorrect, so just remove it, please. Don't mind other header.

Reply:
Removed.